Develop

vaadyak opened the pull request:
- [DEMO LINK](https://vaadyak.github.io/layout_calendar/)

vaadyak committed:
fix mistakes in calendar.scss

vaadyak committed:
add name in readme.md

MaxManis commented on src/index.html:
```diff
@@ -7,6 +7,40 @@
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__box calendar__box--length--29 calendar__box--start-day--sun">
```
I guess you dont need this box element, like you could easily use all of this styles and modifiers for your `calendar ` block)

OleksiiBlezniuk replied:
Agree with @MaxManis

MaxManis commented on src/styles/blocks/variables.scss:
```diff
@@ -0,0 +1,18 @@
+$text-size: 30px;
+$day-size: 100px;
+$gap-size: 1px;
+$calendar-width: $day-size * 7 + $gap-size * 6;
```
```suggestion
$gap-size: 1px;
$week-length: 7;
$gap-counts: $week-length - 1;
$calendar-width: $day-size * $week-length + $gap-size * $gap-counts;
```
Dont use magic numbers)

MaxManis commented on src/styles/calendar.scss:
```diff
@@ -0,0 +1,66 @@
+    text-align: center;
+
+    @include square($day-size);
+    border: 1px solid black;
```
Usually we use vars for colors))

OleksiiBlezniuk replied:
Agree with @MaxManis

MaxManis reviewed (commented):
Great job!) Left some comments!)

vaadyak committed:
Update src/styles/blocks/variables.scss

$gap-size: 1px;
$week-length: 7;
$gap-counts: $week-length - 1;
$calendar-width: $day-size * $week-length + $gap-size * $gap-counts;

Co-authored-by: MaxManis <89999954+MaxManis@users.noreply.github.com>

OleksiiBlezniuk commented on src/index.html:
```diff
@@ -7,6 +7,40 @@
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__box calendar__box--length--29 calendar__box--start-day--sun">
+        <div class="calendar__day calendar__day--day--1"></div>
```
Use pseudoclasses instead modifiers for every element

OleksiiBlezniuk commented on src/styles/blocks/variables.scss:
```diff
@@ -0,0 +1,20 @@
+$calendar-width: $day-size * $week-length + $gap-size * $gap-counts;
+$days: (
+  "mon": 0,
+  "tue": ($day-size + $gap-size),
```
Better to store here only multipliers for day number and move this calculation to function

OleksiiBlezniuk reviewed (approved):
GJ, left few comments for the future, pay attention to it
